### ISSUE TITLE
Compiling Error

### ISSUE THREAD

**Issue by SyedAliZafar** [raspberrypi/tools#89]
i was trying to cross compile my OpenDDS software through it but  i keep getting errors while running the make command  
 i ran configure command which had no errors 
**syedli@ubuntu:~/Desktop/OpenDDS-3.12$ ./configure --target=linux-cross\
[croossscomipiling.txt](https://github.com/raspberrypi/tools/files/1753926/croossscomipiling.txt)
[CRosscompiling Error.txt](https://github.com/raspberrypi/tools/files/1753927/CRosscompiling.Error.txt)

  --target-compiler="/home/syedli/Desktop/tools-master/arm-bcm2708/gcc-linaro-arm-linux-gnueabihf-raspbian-x64/bin/arm-linux-gnueabihf-g++"**

But when i ran make command 
syedli@ubuntu:~ make 
i got the following errors

**Comment by pelwell** [comment]
I don't think this is a problem with the toolchain (which is what these issues are supposed to be about), but if you upload the Makefile somewhere we should be able to work out what is going wrong.
